### PR TITLE
Update android-authentication.mdx

### PR DESCRIPTION
**Pull request by SuQiang2007**
Change android's access document into English and made some improve.

### CODE REVIEW

File: data/android/android-authentication.mdx [hunk 1]
```diff
@@ -1,58 +1,52 @@
-# MarketSDK 安卓接入文档
+# Mirror World Web Android SDK
```
Comment (codebender828) [inline data/android/android-authentication.mdx:1]:
```suggestion
# Mirror World Android SDK
```

File: data/android/android-authentication.mdx [hunk 2]
```diff
@@ -1,58 +1,52 @@
...
 
-最新版本为: 0.1.0 更新时间为: 2022-08-15
+Mirror World's Official TypeScript/JavaScript SDK
```
Comment (codebender828) [inline data/android/android-authentication.mdx:3]:
```suggestion
Mirror World's Official Android SDK  
```

File: data/android/android-authentication.mdx [hunk 3]
```diff
@@ -1,58 +1,52 @@
...
+    public void callback(String result) {
+        Log.i("MyApp","Login successed!");
+    }
+});
 ```
 
-在之后的与 SDK 的互动中，如果用户没有正确地进行过登录，将会自动弹出登录界面引导
-用户登录（但这对用户是不友好的，强烈建议将登录流程集成入 App 的使用流程中）。
+## Full API Documentation
+You can view the documentation for Mirror World SDK for Mobile on [our Official Documentation Site](https://docs.mirrorworld.fun/overview/introduction)
```
Comment (codebender828) [inline data/android/android-authentication.mdx:51]: This link should go to the Android SDK API reference page. Please link to it

**Review by codebender828 — CHANGES_REQUESTED** [review]

File: data/android/android-authentication.mdx (hunk 3 above)
Comment (SuQiang2007) [reply data/android/android-authentication.mdx:51]: Ok,sorry for my mistake. And the link seems can not change here, I will create another pr to fix.

**Pull request closed without merging**